Tryouts/Prototypes/MorganStanley.ComposeUI.Prototypes.NoPluginsPrototype/App.axaml.cs

```diff
@@ -0,0 +1,50 @@
+using Avalonia;
+using Avalonia.Controls.ApplicationLifetimes;
+using Avalonia.Markup.Xaml;
+using NP.Avalonia.Gidon;
+using NP.IoCy;
+using NP.NLogAdapter;
+
+namespace MorganStanley.ComposeUI.Prototypes.NoPluginsPrototype
+{
+    public class App : Application
+    {
+        /// defined the Gidon plugin manager
+        /// use the following paths (relative to the NoPluginsPrototype.exe executable)
+        /// to dynamically load the plugins and services:
+        /// "Plugins/Services" - to load the services (non-visual singletons)
+        /// "Plugins/ViewModelPlugins" - to load view model plugins
+        /// "Plugins/ViewPlugins" - to load view plugins
+        public static PluginManager ThePluginManager { get; } =
```

Comment:
Please avoid this naming convention. Rename to `PluginManager`.

Reply:
done